app/javascript/packages/verify-flow/verify-flow-step-indicator.tsx

```diff
@@ -94,35 +95,35 @@ export function getStepStatus(index, currentStepIndex): StepStatus {
 }
 
 /**
- * Given contextual details of the current flow path, returns explicit statuses which should be used
- * at particular steps.
+ * Given contextual details of the current flow path, returns the relevant flow configuration.
  *
  * @param details Flow details
  *
- * @return Step status overrides.
+ * @return Flow step configuration.
  */
-function getStatusOverrides({
+function getFlowStepsConfig({
+  path,
   addressVerificationMethod,
 }: {
+  path: VerifyFlowPath;
   addressVerificationMethod: AddressVerificationMethod;
-}) {
-  const statuses: Partial<Record<VerifyFlowStepIndicatorStep, StepStatus>> = {};
+}): VerifyFlowConfig {
+  let { steps, mapping } = FLOW_STEP_PATHS[path];
```

Comment:
is this the line the eslint-plugin change was for?

Reply:
With the previous rule enforced, you'd have to do something like...

```ts
let { steps } = FLOW_STEPS_PATH[path];
const { mapping } = FLOW_STEPS_PATH[path];
```

Not only is it kinda garish, I think the general pattern could be problematic in cases such as...

```ts
let { foo } = doSomeExpensiveOrMutativeTask();
const { bar } = doSomeExpensiveOrMutativeTask();
```